front/pages/home/about.tsx

```diff
@@ -206,6 +206,27 @@ const PEOPLE: Record<
     github: "https://github.com/frankaloia",
     linkedIn: "https://www.linkedin.com/in/frank-aloia-39907a12b/",
   },
+  gina: {
+    name: "Gina Kabasakalis",
+    title: "Go To Market",
+    image: "https://ca.slack-edge.com/T050RH73H9P-U08FS7RK45B-0d9a6f5b000d-512",
+    linkedIn: "https://www.linkedin.com/in/ginakabasakalis/",
+    github: "https://github.com/gina-dust/",
+  },
+  apara: {
+    name: "Apara Sivaraman",
+    title: "Business Operations (US)",
+    image: "https://ca.slack-edge.com/T050RH73H9P-U08FS7GNBM3-12d42049c6a8-512",
```

Comment:
@apara-dust when you have a minute if you can update your github profile image so that we can point to it here out of consistency with others in that page :pray: